Sources/Utils/Rng/MobileCoinSeedableRng.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+// swiftlint:disable unused_setter_value
+import Foundation
+
+public class MobileCoinSeedableRng: MobileCoinRng {
+    let _seed: Data32
+
+    var seed: Data32 {
```

Comment:
don't think this dual variable pattern is necessary if the seed is declared as let, means it can only be written to once.

Reply:
Ya, I think I originally had it as var with set/get before I realized that there was no setter on the underlying rust impl.
Will update

Reply:
This is moved into the `MobileCoinChaCha20Rng` class for now, after removing the `MobileCoinSeedableRng`, and the code has been updated to remove the dual varaible.